src/deps/bulkGetShim.js

```diff
@@ -77,7 +77,7 @@ function bulkGet(db, opts, callback) {
     }
 
     // globally-supplied options
-    ['revs', 'attachments', 'binary'].forEach(function (param) {
+    ['revs', 'attachments', 'binary', 'timeout', 'cache'].forEach(function (param) {
```

Comment:
I've discussed this with @nolanlawson, and we're not actually sure if we should be copying `timeout` here, or `ajax`.  I will test further locally.